pulpcore/app/models/task.py

```diff
@@ -287,11 +287,11 @@ def current():
             pulpcore.app.models.Task: The current task.
         """
         try:
-            task_id = get_current_job().id
+            job_id = get_current_job().id
```

Comment:
We're not in control of this name `id` because this is the return type from rq's method `get_current_job()`. So we can't switch this.

Reply:
Ah, gotcha.  Unfortunate, but I understand.  Thanks for clarifying.

Reply:
Also, think of it this way.  get_current_job() returns (presumably) a `Job` object which naturally has an `id` attribute.  So reading like:
```
job = job_id = get_current_job()
job_id = job.id
```
Seems very intuitive to me.

It would also be clearer if the code just used `job.id` instead of `job_id`